unittests/Commands/InterruptSignalAwaiterTests.cpp

```diff
@@ -0,0 +1,54 @@
+//===- unittests/Commands/InterruptSignalAwaiterTests.cpp -----------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "../lib/Commands/InterruptSignalAwaiter.h"
```

Comment:
this is ugly - I'm not sure the best way to include this file to avoid exposing this as public API